src/bin/ion/commands/beta/generate/context.rs

```diff
@@ -52,7 +53,10 @@ pub enum AbstractDataType {
     //    value: Vec<i64>
     // }
     // ```
-    Sequence(String),
+    Sequence {
+        element_type: String,
+        sequence_type: String,
+    },
```

Comment:
(🗺️ PR tour) Changes `Sequence` variant to include information of `sequence_type` ("list" or "sexp") and `element_type`.